### PR TITLE
seroprevalence use case

### PR DESCRIPTION
**Pull request by ruthmccabe**
Seroprevalence use case: essentially the issue of reporting the denominator from neutralisation testing incorrectly thus overinflating seroprevalence estimates, and how this would impact reaching herd immunity

### CODE REVIEW

File: inst/use_cases/seroprevalence.Rmd [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Of the `r 2*positive_elisa` samples sent for further testing, only `r positive_neutralisation` return as positive. For the neutralisation tests, in respect of the ELISA tests, this corresponds to a seroprevalence estimate of `r round(binom.confint(x = positive_neutralisation,n = 2*positive_elisa,method="exact")$mean,3) * 100`% (95% exact binomial confidence intervals `r round(binom.confint(x = positive_neutralisation,n = 2*positive_elisa,method="exact")$lower,3) * 100`% - `r round(binom.confint(x = positive_neutralisation,n = 2*positive_elisa,method="exact")$upper,3) * 100`%), which is notably lower than the seroprevalence obtained under ELISA only. \
```
Comment (joshwlambert) [inline inst/use_cases/seroprevalence.Rmd:29]: I see a "seroprevalence estimate of 6%" for the ELISA and a "a seroprevalence estimate of 8.3%" for the neutralisation test. So I think this sentence should say higher not lower.

**Comment by joshwlambert** [comment]
Thanks @ruthmccabe, I really like this use case. I think it hits the nail of the head of real-world implications for bad/unclear reporting. Although it doesn't have a figure, I don't feel it needs one and the text makes very clear the implications for vaccination and fatality scenarios. 

I think this is ready to be merge and put into the paper. I propose we place it in the _Contextual information and metadata
_ section.

File: inst/use_cases/seroprevalence.Rmd (hunk 1 above)
Comment (ruthmccabe) [reply inst/use_cases/seroprevalence.Rmd]: good catch - thanks! I also changed the figures so they're more different to underline the point

**Comment by ruthmccabe** [comment]
Thanks so much for the positive feedback!

**Comment by joshwlambert** [comment]
Thanks for the updates @ruthmccabe. Are you happy for me to merge this and incorporate into the manuscript?

**Comment by ruthmccabe** [comment]
Sounds good, thanks!

**Pull request merged**